package.json

```diff
@@ -2,25 +2,30 @@
   "name": "agoda-tslint",
   "version": "0.3.1",
   "description": "A set of TSLint rules used on some Agoda projects.",
-  "repository": "https://github.com/agoda-com/agoda-tslint.git", 
+  "repository": "https://github.com/agoda-com/agoda-tslint.git",
   "author": "Maciej Malkowski <maciej.malkowski@agoda.com>",
   "main": "agoda-tslint.json",
   "license": "Apache-2.0",
   "private": false,
   "scripts": {
-    "build": "yarn tslint && tsc",
-    "tslint": "node node_modules/tslint/lib/tslint-cli.js -p tsconfig.json"
+    "build": "yarn lint && tsc",
+    "lint": "tslint -p tsconfig.json",
+    "test": "jest"
   },
   "peerDependencies": {
-    "tslint": "^5.10.0",
-    "typescript": "^2.9.2"
+    "tslint": ">=5.10.0",
+    "typescript": ">=2.9.2"
   },
   "dependencies": {
     "tsutils": "^2.27.2"
   },
   "devDependencies": {
-    "@types/node": "^10.5.2",
-    "tslint": "^5.10.0",
-    "typescript": "^2.9.2"
+    "@types/jest": "^24.0.11",
+    "@types/node": "^11.13.0",
+    "jest": "^24.7.1",
+    "ts-jest": "^24.0.1",
+    "tslint": "^5.15.0",
+    "tslint-config-airbnb": "^5.11.1",
+    "typescript": "^3.4.1"
```

Comment:
Should we consolidate this version with the version in peer-Dependencies? It's weird they are not the same.